devdocs-generator.ps1

```diff
@@ -28,22 +28,32 @@ function Generate-MarkdownFiles($data, $outputDir, $jsonFilePath, $type) {
 
     foreach ($itemName in $data.PSObject.Properties.Name) {
         $itemDetails = $data.$itemName
-        $category = $itemDetails.Category -replace '[^a-zA-Z0-9]', '-' # Sanitize category name for directory
+        $category = $itemDetails.category -replace '[^a-zA-Z0-9]', '-' # Sanitize category name for directory
         $categoryDir = "$outputDir/$category"
 
         # Create the category directory if it doesn't exist
         if (-Not (Test-Path -Path $categoryDir)) {
             New-Item -ItemType Directory -Path $categoryDir | Out-Null
         }
 
-        $filename = "$categoryDir/$itemName.md"
-        $relativePath = "$outputDir/$category/$itemName.md" -replace '^docs/', ''
+        # Remove "WPF", "WinUtil" prefix and category from the name
+        $displayName = $itemName -replace 'WPF', ''
+        $displayName = $displayName -replace 'WinUtil', ''
+        $displayName = $displayName -replace 'Toggle', ''
+        $displayName = $displayName -replace $category, ''
+        $displayName = $displayName -replace 'Features', ''
+        $displayName = $displayName -replace 'Tweaks', ''
+        $displayName = $displayName -replace 'Panel', ''
+        $displayName = $displayName -replace 'Fixes', ''
```

Comment:
```suggestion
        $displayName = $itemName -replace "^WPF($category|WinUtil|Toggle|Features|Tweaks|Panel|Fixes)", ''
```
> `^`: Anchor character (related to position of where you want to match), means the very start of the string (think of it like after newline, or beginning of data stream)
> `(RegExHere)`: Grouping, useful for many things
> `Hello|Goodbye`: the `|` means **or**, so the word you're looking for is Either `Hello` _or_ `Goodbye`

Change from `'` to `"` to make use of string expansion of `$category` variable (telling powershell to return its value, instead of treating it like a literal `$category` string)

Reply:
ohh thanks for the suggestion. is it easy to update the suggestion? bc then I would just commit it throught the button bc I'm lazy..

Reply:
> ohh thanks for the suggestion. is it easy to update the suggestion? bc then I would just commit it throught the button bc I'm lazy..

@MyDrift-user 
.. didn't get what you mean, what needs to be updated in my suggestion?

Reply:
It looks like it got fixed on it's own then, I was not able to merge the suggestion because it said you have to update the commit suggestion because of a push i did after. some time later it worked fine.